app/presenters/recommended_feed_presenter.rb

```diff
@@ -0,0 +1,15 @@
+class RecommendedFeedPresenter < FeedPresenter
+  def name
+    'RecommendedFeed'
+  end
+
+  def to_partial_path
+    "shared/#{name.underscore}"
+  end
+
+  private
+
+  def load_feed_items!
+    @feed_items.each_with_index { |feed_item, index| @feed_items[index] = RecommendedFeedItemPresenter.new(feed_item: feed_item, current_user: current_user) }
+  end
+end
```

Comment:
DRY. Makes use of inheritance, separation of concern, clean views. Our recommended feed presenter is a collection that "wraps" recommended (starred) feed items providing additional functionality to be used in our subsequent views. Other members (#name and #to_partial_path) allows the rails to identify what partials need to be used to render this object.